packages/client-search/src/types/BrowseOptions.ts

```diff
@@ -1,11 +1,11 @@
 import { BrowseResponse } from '.';
-import { ObjectWithObjectID } from './ObjectWithObjectID';
+import { Hit } from './Hit';
 
 export type BrowseOptions<TObject> = {
   /**
    * The callback called for each batch of objects.
    */
-  readonly batch?: (batch: ReadonlyArray<TObject & ObjectWithObjectID>) => any;
+  readonly batch?: (batch: ReadonlyArray<Hit<TObject>>) => any;
```

Comment:
there's no highlighting on browse either, but I think e.g. _rankingInfo can be retrieved

Reply:
See fc268e0eee87b835fe85eeaf0d9225c31e9d5b35.